src/components/ConversationSettings/ListableSettings.vue

```diff
@@ -0,0 +1,131 @@
+<!--
+  - @copyright Copyright (c) 2020 Vincent Petry <vincent@nextcloud.com>
+  -
+  - @author Vincent Petry <vincent@nextcloud.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="app-settings-subsection">
+		<div id="moderation_settings_listable_conversation_hint" class="app-settings-section__hint">
+			{{ t('spreed', 'Defines who can find this conversation without being invited') }}
+		</div>
+		<div>
+			<label for="moderation_settings_listable_conversation_input">{{ t('spreed', 'Visible for') }}</label>
+			<Multiselect id="moderation_settings_listable_conversation_input"
+				v-model="listable"
+				:options="listableOptions"
+				:placeholder="t('spreed', 'Visible for')"
+				label="label"
+				track-by="value"
+				:disabled="isListableLoading"
+				aria-describedby="moderation_settings_listable_conversation_hint"
+				@input="saveListable" />
```

Comment:
Needs to be a bit wider
![Bildschirmfoto von 2020-12-11 09-41-07](https://user-images.githubusercontent.com/213943/101881693-109fb500-3b95-11eb-8c98-fc7d101530c9.png)